src/postamble.js

```diff
@@ -419,7 +419,7 @@ function abort(what) {
   EXITSTATUS = 1;
 
 #if ASSERTIONS == 0
-  throw 'abort(' + what + '). Build with -s ASSERTIONS=1 for more info.';
+  console.error('abort(' + what + '). Build with -s ASSERTIONS=1 for more info.');
```

Comment:
these should be calls to `err` instead of `console.error` (as in some environments that is redirected to another place, and `console.*` may not exist).